controls/aws-foundations-cis-2.5.rb

```diff
@@ -97,8 +97,8 @@
 
   if aws_config_delivery_channel.exists?
     describe aws_config_delivery_channel do
-      its('s3_bucket_name') { should cmp config_delivery_channels[region]['s3_bucket_name'] }
-      its('sns_topic_arn') { should cmp config_delivery_channels[region]['sns_topic_arn'] }
+      its('s3_bucket_name') { should cmp config_delivery_channels[:"#{region}"][:'s3_bucket_name'] }
+      its('sns_topic_arn') { should cmp config_delivery_channels[:"#{region}"][:'sns_topic_arn'] }
     end
```

Comment:
If possible, use the `#{input('name')}` directly rather then assigning it to a local variable in the control. This should be true for all the controls in the profile.

Reply:
Implemented this for the `region` variable. I can work on others when we move to version 1.3